ansible/get_cpulist.sh

```diff
@@ -33,100 +73,207 @@ then
 	exit 1
 fi
 
-dev_numa_node=$(lspci  -s $dev_pci -v | grep -o "NUMA node [0-9]*" | awk '{print $3}')
-numa_cpulist=$(lscpu | grep "NUMA node${dev_numa_node}" | awk '{print $4}')
-first_ht_pair=$(cat /sys/devices/system/cpu/cpu${dev_numa_node}/topology/thread_siblings_list | tr , " ")
-dut_isolated_cpus=$numa_cpulist
+# Make sure the system has the minimum required NUMA resources
+req_numa_count=2
```

Comment:
We can run this on single numa systems.  This was an issue when we first did a lot of work on these tests as the lab system was single numa.  We should support this.  Anyway to add something here for single numa?